Praksa/Core/Entities/Cinema.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Core.Entities
```

Comment:
Newer versions of .NET Core should allow for
`namespace Core.Entities;

public class Cinema {
    //logic goes here
}`
This way you have one less indent in your code, but it is either a personal preference or a practice decided within the team.